packages/admin-next/dashboard/src/v2-routes/customers/customer-create/components/create-customer-form/create-customer-form.tsx

```diff
@@ -1,61 +1,44 @@
-import { zodResolver } from "@hookform/resolvers/zod"
-import { Button, Heading, Input, Text } from "@medusajs/ui"
-import { useAdminCreateCustomer } from "medusa-react"
-import { useForm } from "react-hook-form"
-import { useTranslation } from "react-i18next"
 import * as zod from "zod"
-
-import { Form } from "../../../../../components/common/form"
+import { Button, Heading, Input, Text } from "@medusajs/ui"
 import {
   RouteFocusModal,
   useRouteModal,
 } from "../../../../../components/route-modal"
+import { Form } from "../../../../../components/common/form"
+import { useForm } from "react-hook-form"
+import { useTranslation } from "react-i18next"
+import { zodResolver } from "@hookform/resolvers/zod"
+import { useCreateCustomer } from "../../../../../hooks/api/customers"
 
-const CreateCustomerSchema = zod
-  .object({
-    email: zod.string().email(),
-    first_name: zod.string().min(1),
-    last_name: zod.string().min(1),
-    phone: zod.string().min(1).optional(),
-    password: zod.string().min(8),
-    password_confirmation: zod.string().min(8),
-  })
-  .superRefine(({ password, password_confirmation }, ctx) => {
-    if (password !== password_confirmation) {
-      return ctx.addIssue({
-        code: zod.ZodIssueCode.custom,
-        message: "Passwords do not match",
-        path: ["password_confirmation"],
-      })
-    }
-  })
+const CreateCustomerSchema = zod.object({
+  email: zod.string().email(),
+  first_name: zod.string().min(1),
+  last_name: zod.string().min(1),
+  phone: zod.string().min(1).optional(),
+})
```

Comment:
How would passwords be handled now? Send a reset password link?

Reply:
Yes, we should be able to set the password on customer accounts. That is a separate flow after creating the customer.